docs/configuration/settings.rst

```diff
@@ -507,6 +507,56 @@ dangerous to leave on by default, and must therefore be enabled explicitly.
 Then, issue a `POST` request to the `/__flush__` endpoint to flush all
 the data.
 
+
+.. _configuration-client-caching:
+
+Client caching
```

Comment:
This section remembers me that I duplicated the documentation in kinto and formatted it as a table. We might backport this change directly in cliquet instead, otherwise we would need to update the kinto documentation each time a setting is changed on cliquet.